java/sage/media/sub/SubtitleHandler.java

```diff
@@ -121,7 +134,11 @@ else if (rv.length() == 0 && willDisplay)
         }
       }
     }
-    //System.out.println("Got Subtitle text=" + rv + " subEntryPos=" + subEntryPos);
+    finally
+    {
+      subtitleLock.readLock().unlock();
+    }
+    System.out.println("Got Subtitle text=" + rv + " subEntryPos=" + subEntryPos);
```

Comment:
I went through all of the subtitle classes and applied the the subtitle debug boolean only to things that were not actual errors or warnings.